ChangeLog.d/8482.txt

```diff
@@ -0,0 +1,6 @@
+Default behavior changes
```

Comment:
I'm not sure about this. The behaviour doesn't change in the default config, which has both enabled anyway. I'm not sure what the best category is, I'm inclined to go for "Changes" since it's officially defined as a catch-all.

Reply:
I wasn't sure either, but to me it looked like a behavior change because even thought they were both defined in the default config, they are no more "connected to each other" in the sense that one does not imply the other.
Anyway if you prefer `Changes` that's fine for me ;)

Reply:
“Default behavior changes” is for incompatible changes in documented behavior. It's similar to “API changes”, but we use “API changes” when affected programs typically won't compile anymore, vs “Default behavior changes” when affected programs will still compile but behave differently.

Here there's no change in documented behavior, so no changelog entry is needed. However, it's plausible that people might have relied on the undocumented behavior that enabling CCM auto-enabled CCM\*, so it's ok to have a notice in the changelog to say this won't happen anymore. That would belong under “Changes” since it doesn't really belong anywhere else.